content/docs/state-and-lifecycle.md

```diff
@@ -191,19 +191,19 @@ ReactDOM.render(
 );
 ```
 
-[**Try it on CodePen**](https://codepen.io/gaearon/pen/KgQpJd?editors=0010)
+[**CodePen मा प्रयास गर्नुहोस्**](https://codepen.io/gaearon/pen/KgQpJd?editors=0010)
 
-Next, we'll make the `Clock` set up its own timer and update itself every second.
+अर्को, हामी `Clock` लाई यसको आफ्नै टाइमर सेटअप गर्नेछौं र प्रत्येक सेकेन्डमा आफैं अपडेट गर्ने बनाउनेछौं।
 
-## Adding Lifecycle Methods to a Class {#adding-lifecycle-methods-to-a-class}
+## Classमा लाइफसाइकल मेथड थप्ने विधि {#adding-lifecycle-methods-to-a-class}
 
-In applications with many components, it's very important to free up resources taken by the components when they are destroyed.
+धेरै कम्पोनेंटहरू सहितको Applicationहरूमा कम्पोनेंटहरू नष्ट हुने बित्तिकै कम्पोनेंटहरू द्वारा लिइएको स्रोतहरू खाली गर्न यो धेरै महत्त्वपूर्ण छ।
```

Comment:
Don't you think`स्रोतहरू` gives the meaning `sources` whereas the reader should understand that these are computing resources such as memory/cpu ? I don't really know what to replace this with, but maybe think about what could be better?

Reply:
Yup! Tried thinking of alternatives! Couldn't think of any as of now so used "resources" as fallback